integration-tests/src/test/java/com/datastax/oss/driver/core/cql/BoundStatementCcmIT.java

```diff
@@ -126,19 +127,23 @@ public void setupSchema() {
                 .build());
   }
 
-  @Test(expected = IllegalStateException.class)
+  @Test
   public void should_not_allow_unset_value_when_protocol_less_than_v4() {
     DriverConfigLoader loader =
         SessionUtils.configLoaderBuilder()
             .withString(DefaultDriverOption.PROTOCOL_VERSION, "V3")
             .build();
-    try (CqlSession v3Session = SessionUtils.newSession(ccmRule, sessionRule.keyspace(), loader)) {
-      PreparedStatement prepared = v3Session.prepare("INSERT INTO test2 (k, v0) values (?, ?)");
+    try (CqlSession v3Session = SessionUtils.newSession(ccmRule, loader)) {
+      PreparedStatement prepared =
+          v3Session.prepare(
+              String.format("INSERT INTO %s.test2 (k, v0) values (?, ?)", sessionRule.keyspace()));
```

Comment:
Looked at in isolation it's a bit unclear why this change fixes the problem.  Perhaps add a comment with additional details here, most importantly the error message that you pulled out of Cassandra via the debugger (and optionally where it came from as well).  Might be useful if we find ourselves wanting to add a version bound to this test later on.

Reply:
I suppose it is basically spelled out in the JIRA ticket so if you just want to leave it there I'm good with that.  Maybe a pointer here saying something like "see JAVA-3068 for relevant details about the structure of this test" might not be the worst idea... but given that the ticket spells it out it's certainly not necessary to do a whole lot more here.